CHANGELOG.md

```diff
@@ -16,12 +16,26 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   - `setConfig` return `Ok(false)` when using local config.
 - Modified the repair mode to operate only with a local configuration. Giganto
   now terminates if the repair option is specified without the `-c` flag.
+- The term "source" is replaced with "sensor," resulting in the following major
+  changes:
+  - The `sources` column family in the DB is replaced with `sensors` column
+    family. Running this version of Giganto will migrate the existing data in
+    `sources` column familiy to `sensors` column family.
+  - The `sources` GraphQL API is renamed to `sensors`.
+  - The `sourceId` field in the `export` GraphQL API  is renamed to `sensorId`.
+  - The `source` field in the filter parameters of all GraphQL APIs is changed
+    to `sensor`.
 
 ### Removed
 
 - Removed the GraphQL API `setAckTransmissionCount` as the entire configuration
   is now sent at once when modified through the UI.
 
+### Fixed
+
+- Fixed a missing update to the schema.graphql file necessary for
```

Comment:
```suggestion
- Fixed a missing update to the schema. graphql file necessary for
```

Reply:
It means the file schema.graphql, so I think it should not have space.